src/Stripe.net/Infrastructure/Public/StripeRequest.cs

```diff
@@ -5,15 +5,17 @@ namespace Stripe
     using System.Net.Http;
     using System.Net.Http.Headers;
     using System.Text;
-    using Stripe.Infrastructure;
     using Stripe.Infrastructure.FormEncoding;
 
     /// <summary>
     /// Represents a request to Stripe's API.
     /// </summary>
     public class StripeRequest
     {
+        // Either options or content can be set.
+        // If content is set, it will be used instead of serialized options.
         private readonly BaseOptions options;
+        private readonly string content;
 
         /// <summary>Initializes a new instance of the <see cref="StripeRequest"/> class.</summary>
         /// <param name="client">The client creating the request.</param>
```

Comment:
I assume we're keeping this around for backwards compatility; should we add to the comment that this can not be used to make V2 API calls?

Reply:
In what scenarios would one want to use it in v1?

Reply:
I'm not sure; I assume since its public it was kept in for backwards compatibility (i.e. if a user was creating it directly), but we hardcode the V1 api mode inside this method so in that case if a user tried to make V2 calls with it they would likely fail.